package.json

```diff
@@ -26,6 +26,8 @@
   },
   "dependencies": {
     "chalk": "^1.1.3",
-    "minimist": "^1.2.0"
+    "htmltojsx": "git://github.com/harrison/react-magic.git",
```

Comment:
Ok, sounds good. Manually tested and approved. Merging in. Thanks for the contribution, the project is definitely getting better :)